app/models/temp_user.rb

```diff
@@ -0,0 +1,6 @@
+class TempUser < ActiveRecord::Base
+  has_many :sent_transactions, as: :sender, class_name: 'Transaction'
+  has_many :received_transactions, as: :recipient, class_name: 'Transaction'
+
+  validates :email, presence: true, email: true
```

Comment:
It looks like the email validator also makes the presence validation unnecessary - can we go ahead and remove it?